SeaBattle/Sea.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class Sea
+    {
+        private List<Deck> Decks { get; set; }
+        private List<BaseShip> Ships { get; set; }
+        int SeaWidth;
+        int SeaLenght;
+
+        public Sea()
+        {
+            var Ships = new List<BaseShip>();
```

Comment:
And where does this variable goes then? Do you know what is the scope of this variable?

Reply:
It is useless part of a code which do nothing(dont remember even when I add it, maybe tested something).

Reply:
don't push code for internal testing in the repo. It's a bad practice. so please remove everything that we don't need

Reply:
Got it.

Reply:
don't see any change that I asked for